Readme.md

```diff
@@ -36,38 +36,36 @@ self.addMultiParentGraphQLResolvers({
 
 ## Running Locally
 
-First launch dgraph and load it with the todo schema (and add a todo or two).
```

Comment:
Why are removing the test graphql schema? Is it not worth it to keep here? Can we have another easy setup method so that anyone can try it easily

Reply:
This step is irrelevant.
The readme ask to load a GraphQL schema but the test is using graphql-worker .
So the test is directly a call to a lambda server and the lambda function is doing a concat of 2 fields.
It has nothing to do with the GraphQL Schema deployed.
I think the step was confusing and unnecessary.